### DISCUSSION TITLE
How to swap 's' and '/' key bindings

### DISCUSSION THREAD

**Discussion by uwla** [sxyazi/yazi#1366] [1. Q&A] [answered]
How to swap 's' and '/' default key bindings? Such that the 's' will find next occurrence
and '/' will search for files.

Thanks in advance ;-)

**Comment by sxyazi** [comment, marked as answer]
> '/' will search for files

```
{ on = ["/"], run = "search fd" }
```

> the 's' will find next occurrence

```
{ on = ["s"], run = "find_arrow" }
```

Doc of keymap.toml: https://yazi-rs.github.io/docs/configuration/keymap

**Reply by uwla** [reply]
Thank you!

I actually used the `find` instead of `find_arrow` because the former opens up an input to type while the later does not.
